src/main/java/com/authlete/common/util/CLI.java

```diff
@@ -303,7 +303,7 @@ public void setParameters(String[] args, int from)
                 {
                     case 2:
                         value = pair[1];
-                        // FALLTHROUGH
+                        break;
 
```

Comment:
When pair.length == 2, `key` remains `null`, and this causes `parameters.put(null, value)`. Is this what you intentionally did?